modules/virt-deleting-dvs.adoc

```diff
@@ -5,9 +5,19 @@
 :_content-type: PROCEDURE
 [id="virt-deleting-dvs_{context}"]
 
-= Deleting a data volume
+= Deleting data volumes
 
-You can delete a data volume by using the `oc` command-line interface (CLI).
+You can delete a data volume if needed by using the `oc` command-line interface (CLI), but be aware that doing so also deletes the underlying provisioned volume claim (PVC).
+
+[NOTE]
+====
+When you delete a virtual machine, the data volume it uses is automatically deleted.
```

Comment:
This is true only if the VM has a corresponding dataVolumeTemplates section which references the DV.  It is possible to create a standalone DV and use it with a VM.  If this happens then the DV will not be removed when the VM is deleted.

Reply:
This NOTE was a hanger-on from (I am guessing) a bug that came up likely many years ago. Do you want the behavior you mention in this note to be mentioned in the creating templates topic perhaps? Or is it too much of an edge case? I can also just delete the NOTE, as another option. Let me know what you think.